data/yam/support_images/autoyast_sles15sp3_media_install_textmode_minimal_aarch64.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0"?>
```

Comment:
now there is intermediate /autoyast/ folder and the name of the .xml file doesn't require it.

Reply:
Ok, updated it and VR updated also.